cmd/root.go

```diff
@@ -45,7 +45,11 @@ of Cloud Storage FUSE, see https://cloud.google.com/storage/docs/gcs-fuse.`,
 			if cfgErr != nil {
 				return fmt.Errorf("error while parsing config: %w", cfgErr)
 			}
-			return mountFn(configObj)
+			if bucket, mountpoint, err := populateArgs(args[1:]); err != nil {
+				return fmt.Errorf("error occurred while extracting the bucket and mountpoint: %w", err)
+			} else {
```

Comment:
Nit: remove indented else block (https://google.github.io/styleguide/go/decisions.html#indent-error-flow) and rename mountpoint -> mountPoint